### DISCUSSION TITLE
1 feature 3 solutions

### DISCUSSION THREAD

**Discussion by gilbrotheraway** [mealie-recipes/mealie#4582] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

1) allow importing from websites that block bots as well as social media websites
2) importing recipes from external content i.e text/pdf which many people have 
3) real recipe translation since the current implementation doesn't take into account that most recipe images/notebooks/books would be in the person's native language already, one could translate the recipe page beforehand or directly through openai

### Please provide a concise description of the feature that would resolve your issue.

a simple text box to send it to openai compatible tools

### Please consider and list out some caveats or tradeoffs made in your design decision

none

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [X] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by Kuchenpirat** [comment]
Take a look at #4561
